trieste/acquisition/function.py

```diff
@@ -150,17 +150,30 @@ def acquisition(x: TensorType) -> TensorType:
 
 
 class MinValueEntropySearch(SingleModelAcquisitionBuilder):
-    """
-    Builder for the min-value entropy search acquisition function (an adapted
-    version of max-value entropy search suitiable for function minimisation tasks).
+    r"""
+    Builder for the max-value entropy search acquisition function modified for objective
+    minimisation. :class:`MinValueEntropySearch` estimates the information in the distribution
+    of the objective minimum that would be gained by evaluating the objective at a given point.
+
+    This implementation largely follows :cite:`wang2017max` and samples the objective minimum
+    :math:`y^*` via the empirical cdf :math:`\operatorname{Pr}(y^*<y)`. The cdf is approximated
+    by a Gumbel distribution
+
+    .. math:: \mathcal G(y; a, b) = 1 - e^{-e^\frac{y - a}{b}}
+
+    where :math:`a, b \in \mathbb R` are chosen such that the quartiles of the Gumbel and cdf match.
+    Samples are obtained via the Gumbel distribution by sampling :math:`r` uniformly from
+    :math:`[0, 1]` and applying the inverse probability integral transform
+    :math:`y = \mathcal G^{-1}(r; a, b)`.
```

Comment:
I this info is better here aswell. I put it later on in my PR to mimic the EI function.